apps/passport-server/src/database/queries/telegram/fetchTelegramConversation.ts

```diff
@@ -21,6 +21,21 @@ export async function fetchTelegramConversation(
   return result.rows[0];
 }
 
+export async function fetchTelegramConversationsByChatId(
```

Comment:
@ericzhang98 this is a utility function to get all of the userIds in a chat, will likely be used for when you need to bulk-update the db

Reply:
kk, want to add `telegram_username` field to `TelegramConversation`? or I can do that too

Reply:
doing it rn